### PR TITLE
Settings and Setup Wizard enhancements

### PR DESCRIPTION
**Pull request by arkadiuszzietek**

### CODE REVIEW

File: doofinder-for-woocommerce/readme.txt [hunk 1]
```diff
@@ -126,59 +126,7 @@ Just send your questions to <mailto:support@doofinder.com> and we will try to an
...
-= 1.2.8 =
-Minor compatibility fixes.
-
-= 1.2.7 =
-Updated price retrieving functions. Correctly handle custom attributes added at product level. Multisite support.
-
-= 1.2.6 =
-Handle taxonomy based catalog visibility for WooCommerce 3+.
-
-= 1.2.5 =
```
Comment (carlosescri) [inline doofinder-for-woocommerce/readme.txt:181]: @arkadiuszzietek why did you remove this? 🤔

File: doofinder-for-woocommerce/readme.txt [hunk 2]
```diff
@@ -1,9 +1,9 @@
 === Doofinder for WooCommerce ===
-Contributors: doofinder, chopchoporg
-Tags: search, autocomplete, woo commerce, e-commerce, ecommerce, sales, shop
-Version: 1.2.22
-Requires at least: 5.0
-Tested up to: 5.4.1
```
Comment (carlosescri) [inline doofinder-for-woocommerce/readme.txt:6]: Why this?

File: doofinder-for-woocommerce/readme.txt [hunk 3]
```diff
@@ -1,9 +1,9 @@
...
+Contributors: doofinder
+Tags: search, autocomplete, woocommerce
+Version: 1.2.5
+Requires at least: 4.1
+Tested up to: 4.8
```
Comment (carlosescri) [inline doofinder-for-woocommerce/readme.txt:6]:
```suggestion
Requires at least: 5.0
Tested up to: 5.4.1
```

File: doofinder-for-woocommerce/readme.txt [hunk 4]
```diff
@@ -1,9 +1,9 @@
 === Doofinder for WooCommerce ===
-Contributors: doofinder, chopchoporg
-Tags: search, autocomplete, woo commerce, e-commerce, ecommerce, sales, shop
-Version: 1.2.22
-Requires at least: 5.0
-Tested up to: 5.4.1
+Contributors: doofinder
+Tags: search, autocomplete, woocommerce
+Version: 1.2.5
```
Comment (carlosescri) [inline doofinder-for-woocommerce/readme.txt:4]:
```suggestion
Version: 1.3.0
```

File: doofinder-for-woocommerce/readme.txt [hunk 5]
```diff
@@ -126,59 +126,7 @@ Just send your questions to <mailto:support@doofinder.com> and we will try to an
...
-Minor compatibility fixes.
-
-= 1.2.7 =
-Updated price retrieving functions. Correctly handle custom attributes added at product level. Multisite support.
-
-= 1.2.6 =
-Handle taxonomy based catalog visibility for WooCommerce 3+.
-
-= 1.2.5 =
+= 1.2.4 =
```
Comment (carlosescri) [inline doofinder-for-woocommerce/readme.txt:129]:
```suggestion
= 1.2.22 =
Add support for image sizes.

= 1.2.21 =
Check for existance of post_type before checking for its value.

= 1.2.20 =
Fix separation of multiple value attributes.

= 1.2.19 =
Bugfixes.

= 1.2.18 =
Only load variations for loaded products, instead of all.

= 1.2.17 =
Add some debugging information.

= 1.2.16 =
Export images in thumbnail size instead of full size.

= 1.2.15 =
Add the choice of custom meta fields to settings.
Use variations from parent on feed.

= 1.2.14 =
Fix inconsitence between internal search and JS Layer

= 1.2.13 =
Fix issues.

= 1.2.12 =
Fix issues.

= 1.2.11 =
Fix issues.

= 1.2.10 =
Minor changes.

= 1.2.9 =
Added support for banners in search results.

= 1.2.8 =
Minor compatibility fixes.

= 1.2.7 =
Updated price retrieving functions. Correctly handle custom attributes added at product level. Multisite support.

= 1.2.6 =
Handle taxonomy based catalog visibility for WooCommerce 3+.

= 1.2.5 =
```

File: doofinder-for-woocommerce/doofinder-for-woocommerce.php [hunk 6]
```diff
@@ -134,13 +134,14 @@ public function __construct() {
 						'methods' => ['POST', 'GET'],
 						'callback' => array( $class, 'connect'),
 						'permission_callback' => '__return_true'
-					));
+					),);
```
Comment (carlosescri) [inline doofinder-for-woocommerce/doofinder-for-woocommerce.php:137]:
```suggestion
					));
```

File: doofinder-for-woocommerce/readme.txt (hunk 1 above)
Comment (arkadiuszzietek) [reply doofinder-for-woocommerce/readme.txt]: @carlosescri Hmm i'm not sure why this got removed i didn't mean to remove this.

File: doofinder-for-woocommerce/readme.txt (hunk 2 above)
Comment (arkadiuszzietek) [reply doofinder-for-woocommerce/readme.txt]: @carlosescri Again, i didn't mean to remove this. Not sure how that happened.

File: doofinder-for-woocommerce/readme.txt (hunk 1 above)
Comment (carlosescri) [reply doofinder-for-woocommerce/readme.txt]: OK, don't worry. Everything seems ok, 🤞
